usr/share/rear/lib/borg-functions.sh

```diff
@@ -126,6 +126,9 @@ in Borg repository $BORGBACKUP_REPO on ${BORGBACKUP_HOST:-USB}"
     "${BORGBACKUP_OPT_UMASK[@]}" --exclude-from "$TMP_DIR/backup-exclude.txt" \
     "${borg_dst_dev}${BORGBACKUP_REPO}::${BORGBACKUP_ARCHIVE_PREFIX}_$BORGBACKUP_SUFFIX" \
     "${include_list[@]}"
+    borg_exit_code=$?
+
+    return ${borg_exit_code}
```

Comment:
I think you do not need the added
```
borg_exit_code=$?
return ${borg_exit_code}
```
in the 'borg_create' function because the last command
in that function is `borg create` and
"the exit status of a function is the exit status
 of the last command executed in the body"
(cf. "man bash")
so the exit status of the 'borg_create' function
is the exit status of the `borg create` comamnd.

Reply:
yes, you are right! fixed it